src/api.jl

```diff
@@ -92,8 +148,31 @@ module API
         simflags::String = ""
         )
 
-        exp = join(["simulate", "(", className, ",", "startTime", "=", startTime,",", "stopTime", "=", stopTime,",", "numberOfIntervals", "=", numberOfIntervals,",", "tolerance", "=", tolerance,",", "method", "=", modelicaString(method), ",", "fileNamePrefix", "=", modelicaString(fileNamePrefix), ",", "options", "=", modelicaString(options), ",", "outputFormat", "=", modelicaString(outputFormat), ",", "variableFilter", "=", modelicaString(variableFilter), ",", "cflags", "=", modelicaString(cflags), ",", "simflags", "=", modelicaString(simflags),")"])
-        return sendExpression(omc, exp)
+        exp = join(["simulate", "(", className, ",",
+                                  "startTime", "=", startTime, ","])
+        # There is no default value for stopTime we can provide that behaves like not giving any value and using the stopTime from the experiment annotation...
+        if !isnothing(stopTime)
+            exp *= "stopTime = $stopTime,"
+        end
```

Comment:
This special treatment of `stopTime` is stupid, but I couldn't figure out a way to change this without changing omc itself. The [scripting documentation in the User's Guide](https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/scripting_api.html#simulate) is wrong.

> input Real stopTime = 1.0 "the stop time of the simulation. <default> = 1.0";

The default value isn't `1.0` but the value specified in the experiment annotation. But there is no value one can provide to the scripting call to get the same behavior as not specifying `stopTime`.